project/sites/interchangeni/themes/interchangeni_theme/templates/content/node--opportunity--full.html.twig

```diff
@@ -0,0 +1,116 @@
+ *   - node--type-[type]: The current node type. For example, if the node is an
+ *     "Article" it would result in "node--type-article". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - node--view-mode-[view_mode]: The View Mode of the node; for example, a
+ *     teaser would result in: "node--view-mode-teaser", and
+ *     full: "node--view-mode-full".
+ *   The following are controlled through the node publishing options.
+ *   - node--promoted: Appears on nodes promoted to the front page.
+ *   - node--sticky: Appears on nodes ordered above other non-sticky nodes in
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ */
+#}
+{% set classes = [
+  'node',
+  'node--type-' ~ node.bundle|clean_class,
+  node.isPromoted() ? 'node--promoted',
+  node.isSticky() ? 'node--sticky',
+  not node.isPublished() ? 'node--unpublished',
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+  'clearfix',
+] %}
+
+{% if not page %}
+<article{{ attributes.addClass(classes).setAttribute('id', 'node-' ~ node.id).removeAttribute('role') }}>
+  {% endif %}
+  {{ content.field_opportunity_date }}
+  {{ content.field_manager }}
+  {# {{ kint(content.field_hosting_proforma) }} #}
+  {% if content.field_hosting_proforma['#object'] is defined or content.field_nics_cover_note['#object'] is defined or content.field_partners_cover_note['#object'] is defined or content.field_managers_cover_note['#object'] is defined or content.field_candidate_proforma['#object'] is defined %}
+    <div{{ attributes.addClass('content-field--label-inline').setAttribute('role', 'presentation') }}>
```

Comment:
This is OK, but in my opinion is really a bit too complicated to have in a twig template. I think that I would have been inclined to put this logic in a preprocess function so that you could work out what the contents of the  <li "list-item"> should be and just output that variable.

Reply:
No worries, I will merge it in for now and revisit this late. Thanks